stem/utils.py

```diff
@@ -376,6 +376,47 @@ def create_sigmoid_tiny_expr(start_time: float, dt_slope: float, initial_value:
             return (f"(1 / (1 + e^(-{beta} * (t - {dt_slope} / 2 - {start_time})))) * ({final_value} - {initial_value})"
                     f"+ {initial_value}")
 
+    @staticmethod
+    def create_box_tiny_expr(transition_parameter: float,
```

Comment:
why do we need the box function?

Reply:
to make a step function for the moving load, as tables are not there for the moving load